md/tomcat-bundle.md

```diff
@@ -6,10 +6,10 @@ The Tomcat bundle is a .zip archive that contains the Apache Tomcat JEE applicat
 The Tomcat bundle is a regular .zip archive based on Tomcat zip distribution.
```

Comment:
Not only Bonita BPM Portal but also Engine. Maybe just "prepackaged along with Bonita BPM".
Replace "Bonita configuration folder" with "Platform setup tool".